docs/pages/reference/terraform-provider/resources/installer.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Reference for the teleport_installer Terraform resource
+description: This page describes the supported values of the teleport_installer resource of the Teleport Terraform provider.
+---
+
+{/*Auto-generated file. Do not edit.*/}
+{/*To regenerate, navigate to integrations/terraform and run `make docs`.*/}
+
+
+
+## Example Usage
+
+```hcl
+# Teleport Installer resource
+
+resource "teleport_installer" "example" {
+  version = "v1"
+  metadata = {
+    name        = "example"
+    description = "Example Teleport Installer"
+
+    expires = "2024-10-12T07:20:50Z"
```

Comment:
Does this come from auto-generator? I'm not sure having `expires` field set on the installer resources makes a ton of sense tbh as a part of the example usage.

Reply:
Yes, please never set expiry in TF, the logic is broken and the TF code will stop working once expiry is past.